packages/winston/src/index.ts

```
@@ -0,0 +1,71 @@
+import type {
+  LazyLoggerMessage,
+  Logger as MeshLogger,
+} from '@graphql-mesh/types';
+import type { Logger as WinstonLogger } from 'winston';
+
+function prepareArgs(lazyArgs: LazyLoggerMessage[]) {
+  const flattenedArgs = lazyArgs
+    .flatMap((lazyArg) => (typeof lazyArg === 'function' ? lazyArg() : lazyArg))
+    .flat(Infinity)
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)